extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergBaseInstructions.java

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.iceberg.util;
+
+import io.deephaven.engine.table.TableDefinition;
+
+import java.util.Optional;
+
+public interface IcebergBaseInstructions {
```

Comment:
Is it worth having this base? It's pretty minimal, and maybe not worth the extra code and shared details between read and write?